[2018-06-17 09:01:46] logger_core.ERROR: Не удалось полключиться к базе данных [] []
[2018-06-17 09:01:46] logger_core.ERROR: Не удалось полключиться к базе данных [] []
[2018-06-30 17:20:23] logger_core.ERROR: Не удалось полключиться к базе данных {"host":"localhost"} []
[2018-06-30 17:20:23] logger_core.ERROR: Не удалось полключиться к базе данных {"host":"localhost"} []
[2018-06-30 17:26:45] logger_core.ERROR: Не удалось полключиться к базе данных {"Host":"localhost","User":"root","Password":"root","Database":"hardplan_db"} []
[2018-06-30 17:26:45] logger_core.ERROR: Не удалось полключиться к базе данных {"Host":"localhost","User":"root","Password":"root","Database":"hardplan_db"} []
[2018-06-30 17:27:49] logger_core.ERROR: Не удалось полключиться к базе данных {"Host":"localhost","User":"root","Password":"root","Database":"hardplan_db"} []
[2018-06-30 17:27:49] logger_core.ERROR: Не удалось полключиться к базе данных {"Host":"localhost","User":"root","Password":"root","Database":"hardplan_db"} []
[2018-06-30 17:28:40] logger_core.ERROR: Не удалось полключиться к базе данных {"Host":"localhost","User":"root","Password":"root","Database":"hardplan_db","0":{"Error path":"/var/www/public/db_connect.php"}} []
[2018-06-30 17:28:40] logger_core.ERROR: Не удалось полключиться к базе данных {"Host":"localhost","User":"root","Password":"root","Database":"hardplan_db","0":{"Error path":"/var/www/public/db_connect.php"}} []
[2018-06-30 17:35:46] logger_core.ERROR: Не удалось полключиться к базе данных {"Host":"localhost","User":"root","Password":"root","Database":"hardplan_db","Error path":"/var/www/public/db_connect.php"} []
[2018-06-30 17:35:46] logger_core.ERROR: Не удалось полключиться к базе данных {"Host":"localhost","User":"root","Password":"root","Database":"hardplan_db","Error path":"/var/www/public/db_connect.php"} []
[2018-06-30 17:37:11] logger_core.ERROR: Не удалось полключиться к базе данных {"Host":"localhost","User":"root","Password":"root","Database":"hardplan_db","Error path":"/var/www/public/db_connect.php"} []
[2018-06-30 17:37:11] logger_core.ERROR: Не удалось полключиться к базе данных {"Host":"localhost","User":"root","Password":"root","Database":"hardplan_db","Error path":"/var/www/public/db_connect.php"} []
[2018-06-30 21:42:33] logger_core.ERROR: Не удалось полключиться к базе данных {"Host":"localhost","User":"root","Password":"root","Database":"hardplan_db","Error path":"/var/www/public/db_connect.php"} []
[2018-06-30 21:42:33] logger_core.ERROR: Не удалось полключиться к базе данных {"Host":"localhost","User":"root","Password":"root","Database":"hardplan_db","Error path":"/var/www/public/db_connect.php"} []
[2018-06-30 22:23:13] logger_core.ERROR: Не удалось полключиться к базе данных {"Host":"localhost","User":"root","Password":"root","Database":"hardplan_db","Error path":"/var/www/public/db_connect.php"} []
[2018-06-30 22:23:13] logger_core.ERROR: Не удалось полключиться к базе данных {"Host":"localhost","User":"root","Password":"root","Database":"hardplan_db","Error path":"/var/www/public/db_connect.php"} []
[2018-06-30 22:25:48] logger_core.ERROR: Не удалось полключиться к базе данных {"Host":"localhost","User":"root","Password":"root","Database":"hardplan_db","Error path":"/var/www/public/db_connect.php"} []
[2018-06-30 22:25:48] logger_core.ERROR: Не удалось полключиться к базе данных {"Host":"localhost","User":"root","Password":"root","Database":"hardplan_db","Error path":"/var/www/public/db_connect.php"} []
[2018-06-30 22:27:08] logger_core.ERROR: Не удалось полключиться к базе данных {"Host":"localhost","User":"root","Password":"root","Database":"hardplan_db","Error path":"/var/www/public/db_connect.php"} []
[2018-06-30 22:27:08] logger_core.ERROR: Не удалось полключиться к базе данных {"Host":"localhost","User":"root","Password":"root","Database":"hardplan_db","Error path":"/var/www/public/db_connect.php"} []
